bentoml/adapters/json_input.py

```diff
@@ -110,4 +110,4 @@ def extract_user_func_args(
             except Exception:  # pylint: disable=broad-except
                 err = traceback.format_exc()
                 task.discard(http_status=500, err_msg=f"Internal Server Error: {err}")
-        return (json_inputs,)
+        return json_inputs,
```

Comment:
@akainth015 The formatter we are using is black. It gives different advice than this.